client/src/components/Header.js

```diff
@@ -0,0 +1,22 @@
+import styled from "styled-components";
+
+import { ReactComponent as fin2u } from "../images/fin2u.svg";
+
+export default function Header() {
+  return (
+    <ContainerDiv>
```

Comment:
```suggestion
    <Container>
```

Probably just container is sufficient